README.md

```diff
@@ -14,3 +14,17 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+## Plagarism Statement
+
+All exercises must contain the following statement:
+“I certify that I have listed all sources used to complete this exercise, including the use
+of any Large Language Models. All of the work is my own, except where stated
+otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is
+suspected, charges may be filed against me without prior notice.”
+
+## Note
+I am attempting this from scratch, I will compare it to my last semesters repository and make any corrections needed base on that.
+
+## Answer
+Normal Mergesort has a runtime of $\Theta(nlogn)$, but this Mergesort has extra logic(an extra for loop), which will make it $\Theta(n^{2}logn)$.
```

Comment:
Where is the extra loop, and how do you know that it iterates over all elements?